corehq/apps/export/views.py

```diff
@@ -1546,11 +1547,23 @@ def post(self, request, *args, **kwargs):
             )
             return self.get(request, *args, **kwargs)
 
+        file_size_total = (
+            sum(f.content_length for f in DataFile.objects.filter(domain=self.domain).all()) +
```

Comment:
You can do this in SQL: https://docs.djangoproject.com/en/1.11/topics/db/aggregation/#cheat-sheet